### ISSUE TITLE
Depend on Intl.RelativeTimeFormat directly instead of loading 'relative-time-format' library.

### ISSUE THREAD

**Issue by andrejpavlovic** [catamphetamine/javascript-time-ago#21]
Would it be possible to use this library without [relative-time-format](catamphetamine/relative-time-format) being a required dependency? I would like to load "relative-time-format" as a polyfill only where necessary, and rely on browser native Intl.RelativeTimeFormat when it is available.

**Comment by catamphetamine** [comment]
@andrejpavlovic Yes, there's currently a rewrite pending that will allow optional importing of the polyfill.
Maybe somewhere in October.

**Comment by dandv** [comment]
The RelativeTimeFormat API just got [popular on Twitter](https://twitter.com/wesbos/status/1184893536707207168) :)

**Comment by stof** [comment]
is there any news on this rewrite ?

**Comment by catamphetamine** [comment]
@stof I guess I should.
Maybe in August or September.

**Comment by catamphetamine** [comment]
In version `javascript-time-ago@2.2.0` I've added the ability to use native [`Intl.RelativeTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) and [`Intl.PluralRules`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules) instead of the polyfills: in this case, pass `polyfill: false` option when creating a `TimeAgo` instance.

The `polyfill: false` option still doesn't skip importing the polyfill, so it won't affect the bundle size.
It's of no concern though, because the polyfill is `3.5 kB` in size which is small enough.
https://github.com/tc39/proposal-intl-relative-time#polyfills
